app/src/main/res/values/strings.xml

```diff
@@ -254,6 +254,7 @@
     <string name="recent_tab_text">Recent</string>
     <string name="open_tab_text">Open</string>
     <string name="download_tab_text">Download</string>
+    <string name="favorite_tab_text">Favorites</string>
```

Comment:
spelling of favourite is not consistent here lol

Reply:
fixed